.github/scripts/identify_removable_tests.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+PASSED_TESTS="$(pwd)/passed_tests.txt"
+CURRENT_SKIP_LIST="$(pwd)/current_skip_list.txt"
+REMOVABLE_TESTS="$(pwd)/removable_tests.txt"
+
+while IFS= read -r skip_test; do
+    while IFS= read -r passed_test; do
+        if [[ "${passed_test}"* == "${skip_test}_sh" ]]; then
```

Comment:
This won't work: https://www.shellcheck.net/wiki/SC2203
Globs work only at right-hand side of `==`.